JitsiConference.js

```diff
@@ -542,6 +542,10 @@ JitsiConference.prototype.replaceTrack = function (oldTrack, newTrack) {
         .then(() => {
             if (oldTrack) {
                 this.onLocalTrackRemoved(oldTrack);
+                if(!newTrack && oldTrack.isVideoTrack() && oldTrack.videoType === VideoType.DESKTOP) {
```

Comment:
Are you testing these problems with jitsi-meet or some custom app ?
I imagine this line here fixes display issue which appears also when someone who is sharing the screen video mutes. I'm not comfortable with this change, because the root problem lies elsewhere.